.github/ISSUE_TEMPLATE/wiki-issue.yml

```diff
@@ -0,0 +1,15 @@
+name: Wiki Issue
+description: Something is wrong with the wiki.
+labels: [documentation]
+
+body:
+  - type: markdown
+    attributes:
+      value: Thank you for filing a wiki report!
+  - type: textarea
+    id: description
+    attributes:
+      label: Describe the issue
+      description: Provide a clear and concise description of what the issue is.
```

Comment:
The bug-report template also asks for a "Expected Behavior". If we want to follow that same style for wiki/documentation bug-reports, then we could add a field here for `Expected content`.